jenkins-master/jobs/scripts/workspace/virtualenv/.gitignore

```diff
@@ -0,0 +1,10 @@
+virtualenv.egg-info
```

Comment:
As discussed earlier this week and in our meeting we will completely remove the dependency on that external tool and get it installed globally instead. This work gets being done in [bug 1212922](https://bugzilla.mozilla.org/show_bug.cgi?id=1212922). So I will completely get rid of all of those files in that PR.